src/components/CameraOverlay.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement, useEffect, useRef, useState } from 'react'
```

Comment:
Moved Camera stuff from previous QRCodeOverlay to this shared component, which is used in QRCodeOverlay and ImageSelectionOverlay.